cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlannerTest.java

```diff
@@ -62,7 +62,7 @@
       new ExecutionProposal(new TopicPartition(TOPIC1, 3), 0, _r3, Arrays.asList(_r3, _r2), Arrays.asList(_r2, _r3));
 
   private final ExecutionProposal _partitionMovement1 =
-      new ExecutionProposal(new TopicPartition(TOPIC2, 0), 4, _r0, Arrays.asList(_r0, _r2), Arrays.asList(_r2, _r1));
+      new ExecutionProposal(new TopicPartition(TOPIC2, 0), 1, _r0, Arrays.asList(_r0, _r2), Arrays.asList(_r2, _r1));
```

Comment:
Note: We didn't need to change the PrioritizeLargeReplicaRebalance strategy tests after making this change, because of: https://github.com/linkedin/cruise-control/issues/1183#issuecomment-621460812.

Reply:
Is there any specific reason we want to change the test case here? 
I just test locally, with the bug fix, If we do not change the test, it can still pass.

Reply:
The test cases were missing the "equal partition size" case, which caused the original bug. Adding this test case proves that the fix is working as expected, and it should help to prevent future regressions (if the chaining breaks in the future).